src/main/java/uk/gov/cshr/config/CustomAuthenticationFailureHandler.java

```diff
@@ -11,10 +11,17 @@
 public class CustomAuthenticationFailureHandler implements AuthenticationFailureHandler {
     @Override
     public void onAuthenticationFailure(HttpServletRequest request, HttpServletResponse response, AuthenticationException exception) throws IOException, ServletException {
-        if (exception.getMessage().equals("User account is locked")) {
-            response.sendRedirect("/login?error=locked");
-        } else {
-            response.sendRedirect("/login?error=failed");
+        String exceptionMessage = exception.getMessage();
```

Comment:
I am a bit suspicious about this.  Is there a better way of doing this other than using the exception message?
(As it is controlled by Spring).

Reply:
Might be best to use the type

Reply:
This customer failure handler class and the `exception.getMessage()` was already in place before my updates for 'account blocked'. I'm going to keep it in for now as this is already quite a significant diff and don't want to rework everything in identity-service.